### ISSUE TITLE
Support unpatched cross / add instructions for running custom `cross`

### ISSUE THREAD

**Issue by CraftSpider** [tectonic-typesetting/tectonic#1035]
While debugging my PR, I wanted to try running the cross tests locally. This turns out to be more complicated than using the default `cross test`, which I figured out after trying to hunt down a 'missing `HOST_GID` variable' error when running the command. It would be nice if there was at least a nicer error for this case, and either support for unpatched `cross`, or instructions on how best to run the patched version locally.

**Comment by CraftSpider** [comment]
Update: I got it to work by doing the following:
- `sudo apt-get install qemu-user-static`
- `docker run --rm -v $(pwd):/work:rw,Z tectonictypesetting/ttcross:latest`
- `rustup override stable`
- `./cross {build, test} --target mips-unknown-linux-gnu --release --all`

Would be nice to have this documented in the future, but for anyone else who runs into this issue, those are the necessary steps to run the tests locally.

**Comment by pkgw** [comment]
Yes, this is another area where the documentation is lacking. It's also been a while since I looked at the infrastructure here; maybe with another look, the custom `cross` tool could be avoided at this point.

**Comment by CraftSpider** [comment]
Trying to see if we can do this: #1234
